### DISCUSSION TITLE
Verify cost estimations through benchmarks

### DISCUSSION THREAD

**Discussion by silvanheller** [vitrivr/cottontaildb#124] [General]
Cottontail makes certain decisions based on cost estimations (e.g. whether to take an index on an id column or to perform a table scan). It would be great to verify these cost estimations empirically through benchmarks, especially with a look towards LSC, where we have relational metadata we want to query efficiently.

**Comment by ppanopticon** [comment]
It would be helpful if you were a bit more specific about the types of queries you are performing so that we can include these in the tests. And do you have example data that comes with the queries?
